### DISCUSSION TITLE
how to get requestheader values of OIDCProviderAuthorizationEndpoint

### DISCUSSION THREAD

**Discussion by RamyaAyyanan** [OpenIDC/mod_auth_openidc#862] [Q&A] [answered]
Hi ,
during sso authentication, our identityprovider used to pass sessionid in request header of authorizationendpoint. i need to get the session value to kill that session at identityprovider side. could you please advise how to take it?

**Comment by zandbelt** [comment, marked as answer]
that seems like an identity provider question that does not have to do with mod_auth_openidc; it is non-standards compliant anyhow
